CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
-## v1.0.1
-### 2024-04-05
+# Changelog
+
+## [v1.1.0] - 2024-08-29
+### Changed
+
+- [[DAS-1934](https://bugs.earthdata.nasa.gov/browse/DAS-1934)]
+  Input parameters that include both both resolutions (`xres` and `yres`) and
+  dimenions (`height` and `width`) no longer always raises an exception. An
```

Comment:
Turbo nit: I think it should be "raise" ("Input parameters that... no longer always raises")